modules/MinimumSpanningTree/cover.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
This script is a workaround so that I can view code coverage in NetBeans while developing this plugin. I'm not sure it needs to be included. What do you think?